src/components/Header/index.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import Header, { HeaderProps } from './index';
+import React from 'react';
+import { StoryObj } from '@storybook/react';
+import { MultiValue, SelectInstance } from 'react-select';
+import { mockFetchData } from 'util/mockFetchData';
+import { filterByStringProp } from 'util/mockFetchData/filterData';
+import { mockOptions } from '../SmartInput/mockData';
+import { SmartInputProps, OptionProps } from '../SmartInput';
+import cbiLogo from './CBI-logo-white.svg';
+
+export const Primary: StoryObj<HeaderProps> = {
+  args: {
+    showLogo: true,
+    logoImg: cbiLogo,
+  },
+  argTypes: {
+    onSearchClick: {
+      action: 'onSearchClick',
+    },
+    onAdvancedSearchClick: {
+      action: 'onAdvancedSearchClick',
+    },
+    smartInputProps: {
+      onChange: {
+        action: 'smartInputProps/onChange',
+      },
+      onClear: {
+        action: 'smartInputProps/onClear',
+      },
+      onMenuClose: {
+        action: 'smartInputProps/onMenuClose',
+      },
+      onBlur: {
+        action: 'smartInputProps/onBlur',
+      },
+      onFocus: {
+        action: 'smartInputProps/onFocus',
+      },
+      onKeyDown: {
+        action: 'smartInputProps/onKeyDown',
+      },
+      onOptionClick: {
+        action: 'smartInputProps/onOptionClick',
+      },
+    },
+  },
+  render: ({ showLogo, logoImg, onSearchClick, onAdvancedSearchClick }): JSX.Element => {
+    const [searchItems, setSearchItems] = React.useState<MultiValue<OptionProps>>();
+    const [options, setOptions] = React.useState<OptionProps[]>([]);
+    const ref = React.useRef<SelectInstance<OptionProps, true>>(null);
+    const [inputValue, setInputValue] = React.useState('');
+
+    const onChange = () => {};
+    const onClear = () => {};
+    const onMenuClose = () => {};
+    const onBlur = () => {};
+    const onFocus = () => {};
+    const onOptionClick = () => {};
+    const onKeyDown = () => {};
```

Comment:
Is there a reason we would define all of these here, just adding the functions directly as default parameters?

Reply:
no valid reason really, this is just how i built the story initially, but you're right, defaulted all of these to `noop`